docs/source/user/nalu_run/nalu_inp.rst

```diff
@@ -999,141 +999,7 @@ Time-step Control Options
-   
-.. inpfile:: actuator.simStart
-
-   Flag indicating whether the simulation starts from scratch or restart. ``simStart`` takes on one of three values:
-
-   * ``init`` - Use this option when starting a simulation from `t=0s`.
-   * ``trueRestart`` - While OpenFAST allows for restart of a turbine simulation, external components like the Bladed style controller may not. Use this option when all components of the simulation are known to restart.
-   * ``restartDriverInitFAST`` - When the ``restartDriverInitFAST`` option is selected, the individual turbine models start from `t=0s` and run up to the specified restart time using the inflow data stored at the actuator nodes from a hdf5 file. The C++ API stores the inflow data at the actuator nodes in a hdf5 file at every OpenFAST time step and then reads it back when using this restart option. This restart option is especially useful when the glue code is a CFD solver.
-   
-.. inpfile:: actuator.t_start
-   
-   Start time of the simulation
-
-.. inpfile:: actuator.t_end
-
-   End time of the simulation. ``t_end`` <= ``t_max``
-
-.. inpfile:: actuator.t_max
-
-   Max time of the simulation
-
-
-.. note::
-
-   ``t_max`` can only be set when OpenFAST is running from `t=0s` and ``simStart`` is ``init``. ``t_max`` can not be changed on a restart. OpenFAST will not be able to run beyond ``t_max``. Choose ``t_max`` to be large enough to accomodate any possible future extensions of runs. One can change ``t_start`` and ``t_end`` to start and stop the simulation any number of times as long as ``t_end`` <= ``t_max``.
-
-.. inpfile:: actuator.dt_fast
-
-   Time step for OpenFAST. All turbines should have the same time step.
-
-.. inpfile:: actuator.n_every_checkpoint
-
-   Restart files will be written every so many time steps   
-
-**Turbine specific input options**
-
-.. inpfile:: actuator.turbine_base_pos
-
-   The position of the turbine base for actuator-line simulations
-
-.. inpfile:: actuator.num_force_pts_blade
-   
-   The number of actuator points along each blade for actuator-line simulations   
-   
-.. inpfile:: actuator.num_force_pts_tower
-
-   The number of actuator points along the tower for actuator-line simulations.
-  
-.. inpfile:: actuator.epsilon
-
-   The spreading width :math:`\epsilon` in the Gaussian spreading function in the `[chordwise, spanwise, chord normal]` coordinate system to spread the forces from the actuator point to the nodes. Nalu currently only supports an isotropic Gaussian spreading function and uses only the value in the first component along the `chordwise` direction.
-   
-.. inpfile:: actuator.restart_filename
-
-   The checkpoint file for this turbine when restarting a simulation
-   
-.. inpfile:: actuator.FAST_input_filename
-
-   The FAST input file for this turbine
-  
-.. inpfile:: actuator.turb_id
-
-   A unique turbine id for each turbine
+.. include:: ./turbine_modeling.rst
```

Comment:
A large section was removed from the docs, but the equivalent file `turbine_modeling.rst` was not committed to the repository.

Reply:
This file has now been added to the repository.